packages/browser/src/index.ts

```diff
@@ -30,8 +30,13 @@ export type {
 
 export { replayCanvasIntegration } from '@sentry-internal/replay-canvas';
 
-export { feedbackIntegration } from './feedback';
-export { feedbackAsyncIntegration } from './feedbackAsync';
+import { feedbackAsyncIntegration } from './feedbackAsync';
+import { feedbackSyncIntegration } from './feedbackSync';
+export {
+  feedbackAsyncIntegration,
+  feedbackSyncIntegration,
+  feedbackSyncIntegration as feedbackIntegration
+};
```

Comment:
NPM users will have both strategies to pick from, but `feedbackIntegration` is the name we have all over the docs, so that's available too using the Sync strat.